libs/RobotKit/tests/RobotController_test.h

```diff
@@ -126,6 +126,16 @@ class RobotControllerTest : public testing::Test
 		EXPECT_CALL(mock_motor_right, stop());
 	}
 
+	void expectedCallsStopActuators()
+	{
+		EXPECT_CALL(mock_ledkit, stop).Times(AtLeast(1));
+		EXPECT_CALL(mock_videokit, stopVideo).Times(AtLeast(2));
+		EXPECT_CALL(mock_motor_left, stop()).Times(AtLeast(2));
+		EXPECT_CALL(mock_motor_right, stop()).Times(AtLeast(2));
```

Comment:
unrelated question: if MotionKit is running, will this really stop the actuators? 

RobotController should also stop MotionKit in this case, no?